### PR TITLE
Contaminated Milk 2015 Dec Bronze

### PR DESCRIPTION
**Pull request by jessechoe10**

### CODE REVIEW

**Comment by bqi343** [comment]
https://usaco.guide/general/contributing#adding-a-solution

> Then add the solution ID (usaco-690) as the last argument of the Problem constructor in the corresponding module

_Originally posted by @bqi343 in https://github.com/cpinitiative/usaco-guide/issues/454#issuecomment-735988708_

**Comment by bqi343** [comment]
Also, it needs to deploy successfully.

File: solutions/USACO dec15-contaminated-milk.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+In this problem, we are asked to find how many people are sick based on what type of milk each person drank, when they drank it, and when a person got sick. To solve this problem, 
+we must find the types of milk that could (potentially) make a person sick. To do this, we must check if a person drank milk type X before they became sick. Therefore, we loop over
+every sick person and store the possible milk types in a 2D array of integers. Then, we loop over all the possible milk types for every sick person and check for commonalities. If
+there is consistency in the possible milk types (every sick person), then we add that to an array and consider it one of the possible types of milk that would make someone sick.
+After this, we check how many people drank one of the possible milk types (consistent with each sick person), and print the number of people who were sick.
+
+<LanguageSection>
+
+  <CPPSection>
```
Comment (bqi343) [inline solutions/USACO dec15-contaminated-milk.mdx:18]: Don't use tabs before these, it might break the build.

File: solutions/USACO dec15-contaminated-milk.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  
+    #include <bits/stdc++.h>
```
Comment (bqi343) [inline solutions/USACO dec15-contaminated-milk.mdx:20]: Use ` ```cpp ` around your code, don't indent the entire code block.

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/2_Bronze/Intro_Complete.mdx [hunk 3]
```diff
@@ -73,8 +73,6 @@ export const problems = {
 		), // 98.76
 		new Problem('Bronze', 'Why Did the Cow Cross the Road II', '712', 'Normal', false, ['Nested Loop'], 'usaco-712'), // 89.6
 		new Problem('Bronze', 'Guess the Animal', '893', 'Hard',false, [], ''),
-		new Problem('Bronze', 'Contaminated Milk', '569', 'Hard', false, ['Nested Loop'], 'usaco-569'),
```
Comment (Farrius) [inline content/2_Bronze/Intro_Complete.mdx]: I think this is not intended.

**Review by Farrius — CHANGES_REQUESTED** [review]

File: content/2_Bronze/Intro_Complete.mdx [hunk 4]
```diff
@@ -73,8 +73,6 @@ export const problems = {
...
-		new Problem('Bronze', 'Cowntact Tracing', '1037', 'Hard', true, [], ''),
```
Comment (Farrius) [inline content/2_Bronze/Intro_Complete.mdx]: Also this.

File: content/2_Bronze/Intro_Complete.mdx [hunk 5]
```diff
@@ -91,7 +91,7 @@ export const problems = {
 			[],
 			'Observe that a lot of placements of vertical or horizontal fences are equivalent. Specifically, a fence should be adjacent to a cow, so loop over all $N ^ 2$ unique placements of a vertical and horizontal fence. This idea is similar to Coordinate Compression.'
 		), // 82.79
-		new Problem('Bronze', 'Contaminated Milk', '569', 'Very Hard', false, [], ''),
+		new Problem('Bronze', 'Contaminated Milk', '569', 'Very Hard', false, [], 'usaco-569'),
```
Comment (darren-yao) [inline content/2_Bronze/Intro_Complete.mdx:94]: Why is this change necessary?

File: solutions/USACO dec15-contaminated-milk.mdx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+using namespace std;
+using ll = long long;
+using vl = vector<ll>;
+using pl = pair<ll, ll>;
+using vpl = vector<pl>;
+using vc = vector<char>;
+using vs = vector<string>;
+using str = string;
+
+#define FastIO ios_base::sync_with_stdio(0); cin.tie(0)
```
Comment (darren-yao) [inline solutions/USACO dec15-contaminated-milk.mdx:31]: if possible, solutions should use as few macros as possible.

**Review by darren-yao — COMMENTED** [review]
I left a couple of comments. @jessechoe10

File: content/2_Bronze/Intro_Complete.mdx (hunk 5 above)
Comment (bqi343) [reply content/2_Bronze/Intro_Complete.mdx:94]: to include the internal solution?

File: solutions/USACO dec15-contaminated-milk.mdx (hunk 6 above)
Comment (jessechoe10) [reply solutions/USACO dec15-contaminated-milk.mdx:31]: Oops, I'm kinda used to writing with template LOL. Should I convert to without the macros?

Comment (darren-yao) [reply solutions/USACO dec15-contaminated-milk.mdx:31]: Preferably, yes.

**Review by darren-yao — APPROVED** [review]
approved
